### ISSUE TITLE
celery & channels conflict

### ISSUE THREAD

**Issue by wsf1990** [django/channels#1344]
I'm using the latest celery and channels.But I found that when I use channels,My celery can't receive task.When I close the channels, Celery works well.Dose channels and celery has something conflict?

**Comment by wsf1990** [comment]
My channels config is:
```
CHANNEL_LAYERS = {
    'default': {
        'BACKEND': 'channels_redis.core.RedisChannelLayer',
        'CONFIG': {
            "hosts": [("redis://:*@127.0.0.1:6379/4")],
        },
    },
}
```

**Comment by onegreyonewhite** [comment]
@wsf1990 and what was problem?

**Comment by wsf1990** [comment]
1. I upload file in django, and after upload I use celery task to process this file.When I use channels, django rasied a 413 error.At beginning, I thought this is caused by celery task not work.When I add `DATA_UPLOAD_MAX_MEMORY_SIZE` to settings,the upload work well.
2. I use celery to listen some work when the app start.When I use channels I found the work was started many times and caused no thread to start other task.So I said when use channels can't start celery task.Now I start this work just when the app was started by celery.And this issue was resolved.
